packages/block-library/src/classic/convert-to-blocks-button.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { BlockControls } from '@wordpress/block-editor';
+import { Toolbar, ToolbarButton } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { rawHandler, serialize } from '@wordpress/blocks';
+
+const ConvertToBlocksButton = ( { clientId } ) => {
+	const { replaceBlocks } = useDispatch( 'core/block-editor' );
+	const block = useSelect(
+		( select ) => {
+			return select( 'core/block-editor' ).getBlock( clientId );
+		},
+		[ clientId ]
+	);
+
+	return (
+		<BlockControls>
```

Comment:
It's not really standard to put the `BlockControls` of a block in a separate component like this. I'd keep the `ToolbarButton` here, but move the `BlockControls` and `Toolbar` back to the `edit.js`.

Reply:
Good point - done.